### ISSUE TITLE
pretty-format-ini broken in v2.0.0

### ISSUE THREAD

**Issue by lcnittl** [macisamuele/language-formatters-pre-commit-hooks#99]
It seems that since the ini formatter is broken since including `v2.0.0`.

This file for example:

```ini
[   flake8  ]



max-line-length    =         88

extend-ignore          =
                                         E203
```

passes, not being reformatted.
In `v1.6.1` this file was reformatted to:

```ini
[   flake8  ]
max-line-length = 88
extend-ignore =
	E203
```

Cheers, lcnittl

**Comment by macisamuele** [comment]
@lcnittl  thanks for the report

I think that the regression was introduced in #45
I'm not yet 100% sure on the path forward, probably `iniparse` library has some toggle that we might use a different parser as suggested in #26 
Need to verify a bit more

**Comment by macisamuele** [comment]
Fixed in #113
